test/compress/harmony.js

```diff
@@ -183,8 +183,9 @@ import_statement: {
         import { Bar, Baz } from 'lel';
         import Bar, { Foo } from 'lel';
         import { Bar as kex, Baz as food } from 'lel';
+        import * from 'lel';
     }
-    expect_exact: "import\"mod-name\";import Foo from\"bar\";import{Bar,Baz}from\"lel\";import Bar,{Foo}from\"lel\";import{Bar as kex,Baz as food}from\"lel\";"
+    expect_exact: "import\"mod-name\";import Foo from\"bar\";import{Bar,Baz}from\"lel\";import Bar,{Foo}from\"lel\";import{Bar as kex,Baz as food}from\"lel\";import*from\"lel\";"
```

Comment:
Nevermind the array form - it's a single line. Just a single quoted string will suffice.